weedcoco/importers/mask.py

```diff
@@ -163,7 +163,7 @@ def _image_name_to_mask(name):
         warnings.warn(
             f"{len(categories)} categories defined, but only "
             f"{len(categories_found)} of these are present in masks. "
-            f"Missing are {missing_category_colors}"
+            f"These categories were not found: {missing_category_colors}"
```

Comment:
Both this change and the one at 118 are great improvements in the clarity of the messages to users.

Reply:
> Both this change and the one at 118 are great improvements in the clarity of the messages to users.

They were both your explicit contributions! :D